implementations/micrometer-registry-otlp/src/main/java/io/micrometer/registry/otlp/OtlpMeterRegistry.java

```diff
@@ -393,10 +394,11 @@ Metric writeHistogramSupport(HistogramSupport histogramSupport) {
                     .addExplicitBounds(isTimeBased ? countAtBucket.bucket(getBaseTimeUnit()) : countAtBucket.bucket());
                 histogramDataPoint.addBucketCounts((long) countAtBucket.count());
             }
-            metricBuilder.setHistogram(io.opentelemetry.proto.metrics.v1.Histogram.newBuilder()
-                .setAggregationTemporality(otlpAggregationTemporality)
-                .addDataPoints(histogramDataPoint));
-            return metricBuilder.build();
+        }
+        // else treat as single bucket histogram
+        else {
+            histogramDataPoint.addAllExplicitBounds(Collections.emptyList());
```

Comment:
Unless I'm missing something, this line doesn't serve any purpose. Adding an empty list won't effectively change anything, right?